hazelcast/src/test/java/com/hazelcast/quorum/cache/CacheQuorumListenerTest.java

```diff
@@ -240,12 +234,13 @@ public void onChange(QuorumEvent quorumEvent) {
         config.addQuorumConfig(quorumConfig);
         TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory(3);
         HazelcastInstance instance1 = factory.newHazelcastInstance(config);
-        HazelcastInstance instance2 = factory.newHazelcastInstance(config);
+        factory.newHazelcastInstance(config);
         HazelcastServerCachingProvider cachingProvider = HazelcastServerCachingProvider.createCachingProvider(instance1);
         Cache<Object, Object> cache = cachingProvider.getCacheManager().getCache(cacheName);
         try {
             cache.put(generateKeyOwnedBy(instance1), 1);
         } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Don't print stack traces. Either it is a failure and the test should be aborted; or it is something which should happen and then you also don't want to see the stacktrace.